pom.xml

```diff
@@ -44,6 +44,8 @@
         <artifactId>gatling-maven-plugin</artifactId>
         <version>${gatling-maven-plugin.version}</version>
         <configuration>
+          <!-- Default folder when running the recorder with the 'gatling:recorder' goal: -->
+          <simulationsFolder>${project.basedir}/src/test/java</simulationsFolder>
```

Comment:
The Maven plugin currently defaults to `src/test/scala` (which we should probably also change to default to Java).

Note that the `simulationsFolder` value in `recorder.conf` is never used when running with `mvn gatling:recorder` (because the Maven config key overrides the config file key, and it is always set since it has a default value in the plugin code).

Reply:
> The Maven plugin currently defaults to src/test/scala (which we should probably also change to default to Java).

We can do that. But can't we use the project's test sources directory?

Reply:
Probably a good idea, yes.

The only doubt I have is that in the code, `MavenProject#getTestCompileSourceRoots()` returns a list of paths, while in the POM we can only set one single path with `<testSourceDirectory>`... I did some tests and I have no idea how this list could ever actually contain several elements. 🤷 I'd say we use that, and simply include a fallback to the hardcoded `src/test/java` in case we get several test source roots.

Reply:
gatling/gatling-maven-plugin#109

Reply:
@guilgaly Remove this line then?